packages/hash/frontend/next.config.js

```diff
@@ -34,6 +34,10 @@ process.env.NEXT_PUBLIC_BLOCK_BASED_ENTITY_EDITOR =
 // This allows the frontend to generate the graph type IDs in the browser
 process.env.NEXT_PUBLIC_FRONTEND_DOMAIN = process.env.FRONTEND_DOMAIN;
 
+process.env.NEXT_PUBLIC_SENTRY_DSN = process.env.SENTRY_DSN;
```

Comment:
I had to bump Next.js from 12.2 to 12.3 for this to work.

Before the bump, `NEXT_PUBLIC_SENTRY_DSN` was equal to `"undefined"` instead of `undefined` (likely because of a Next.js bug).

This broke Playwright: https://github.com/hashintel/hash/actions/runs/3292255307/jobs/5427399664#step:8:20